pkg/pillar/devicenetwork/dnc.go

```diff
@@ -675,6 +683,11 @@ func IngestPortConfigList(ctx *DeviceNetworkContext) {
 	log.Infof("Initial DPCL %v", storedDpcl)
 	var dpcl types.DevicePortConfigList
 	for _, portConfig := range storedDpcl.PortConfigList {
+		// Clear the errors from before reboot and start fresh.
+		for i := 0; i < len(portConfig.Ports); i++ {
+			portPtr := &portConfig.Ports[i]
+			portPtr.Reset()
```

Comment:
This reads as if you are resetting the port, not clearing the TestResults. Better to  name the TestResults member function Clear or RecordClear

Reply:
Will rename it as Clear.